package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "budgetify",
+  "version": "1.0.0",
+  "description": "Web application for Exadel Internship project. It is kinda online bank",
+  "main": "server.js",
+  "scripts": {
+    "dev": "nodemon src/server.js",
+    "start": "node src/server.js",
+    "lint": "eslint --ext .js src"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/asadbek2021/piggy-bank.git"
+  },
+  "keywords": [],
+  "author": "Asadbek",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/asadbek2021/piggy-bank/issues"
+  },
+  "homepage": "https://github.com/asadbek2021/piggy-bank#readme",
+  "dependencies": {
+    "bcrypt": "^5.0.1",
+    "cors": "^2.8.5",
+    "dotenv": "^16.0.0",
+    "express": "^4.17.3",
+    "express-handlebars": "^6.0.2",
+    "express-validator": "^6.14.0",
+    "jsonwebtoken": "^8.5.1",
+    "passport": "^0.5.2",
+    "passport-jwt": "^4.0.0",
+    "uuid": "^8.3.2",
+    "winston": "^3.6.0"
+  },
+  "devDependencies": {
+    "eslint": "^8.10.0",
+    "eslint-config-airbnb": "^19.0.4",
+    "eslint-config-airbnb-base": "^15.0.0",
+    "eslint-config-node": "^4.1.0",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-plugin-import": "^2.25.4",
+    "eslint-plugin-jsx-a11y": "^6.5.1",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-prettier": "^4.0.0",
+    "eslint-plugin-react": "^7.28.0",
```

Comment:
react ?

Reply:
It has been installed as:
```npm i -D eslint prettier eslint-plugin-prettier eslint-config-prettier eslint-plugin-node eslint-config-node```
```npx install-peerdeps --dev eslint-config-airbnb```
```eslint --init```

I don't know why react plugins are there. I have done as the lector showed. How should I install plugins only for node?